packages/shared/src/components/post/AuthorOnboarding.tsx

```diff
@@ -39,10 +38,9 @@ export function AuthorOnboarding({
         <li>Gain reputation points by earning upvotes on articles you wrote</li>
       </ol>
       <div
-        className="grid grid-flow-col gap-x-4 mt-6"
+        className="grid grid-flow-col gap-x-4 mt-6 max-w-[4.625rem]"
```

Comment:
I think the `sizeN` has a multiplier of `0.25`.
```suggestion
        className="grid grid-flow-col gap-x-4 mt-6 max-w-[18.5rem]"
```